Update Cost Optimization tool page (#1742)
giantswarm/docs: src/content/advanced/cost-optimization/tools/index.md

    - [AWS CloudWatch data source integration](https://grafana.com/docs/grafana/latest/datasources/cloudwatch/#iam-policies)
    - [Azure Monitor data source integration](https://grafana.com/grafana/plugins/grafana-azure-monitor-datasource/)
- [Goldilocks](https://github.com/FairwindsOps/goldilocks), a tool for refine and discover the right application resource settings

> - There are a lot of payed solutions offering dashboard oriented services that track costs of Kubernetes Applications or help to reduce Cloud costs like:

Reviewer: ```suggestion
- There are multiple paid solutions offering dashboard-oriented services to track Kubernetes application costs or help to reduce cloud costs. For example:
```
